dist-git.spec

```diff
@@ -18,6 +18,7 @@ Source0:        %{name}-%{version}.tar.gz
 BuildArch:      noarch
 
 BuildRequires:  systemd
+BuildRequires:  python3-devel
```

Comment:
Do we actually need this BuildRequires if we don't use stuff like `%py3_build` or `%py2_build`?

Reply:
We do, because `%{python3_sitelib}` is provided by that package.

Reply:
Btw. I just realized, don't we want to use something like `%py2_build`, `%py3_build`, `%py2_install`, `%py3_install`? Because I suspect especially the build macros might be useful for pregenerating .pyc files.